omod/src/main/resources/apps/ugandaemr_homepage_app.json

```diff
@@ -253,6 +253,36 @@
       "requiredPrivilege": "App: coreapps.systemAdministration"
     }
   ]
-}
-
+},{
+  "id": "ugandaemr.cohortExist",
+  "description": "Patient Exist Cohort Page",
+  "order": 41,
+  "extensions": [
+    {
+      "id": "coreapps.activeVisitsHomepageLink",
+      "extensionPointId": "systemAdministration.apps",
+      "type": "link",
+      "label": "Exist from Cohort Group",
+      "url": "ugandaemr/PatientExitCohort.page",
+      "icon": " icon-cog",
+      "requiredPrivilege": "App: coreapps.systemAdministration"
+    }
+  ]
+},
+  {
+    "id": "ugandaemr.facility.dashboard.link",
```

Comment:
This code was moved to the reports-module. But it's re-appearing here

Reply:
Done